templates/vue/src/components/Lightbox/media/MultiContentMedia/index.vue

```diff
@@ -17,17 +17,44 @@
         <completed-icon :node="node" class="mx-2" />
       </h1>
     </header>
-    <multi-content-rows
-      v-if="node.presentationStyle"
-      :dimensions="dimensions"
-      :node="node"
-      :rowId="rowId"
-      :context="context"
-      :level="level"
-      @load="$emit('load')"
-      @change-row="changeRow"
-      @complete="complete"
-    />
+    <template v-if="node.presentationStyle">
+      <div v-if="node.presentationStyle === 'unit'">
+        {{
+          node.childOrdering.length
+            ? "The content of this unit has not been unlocked yet."
+            : "There is no content in this node."
```

Comment:
Let's avoid using the word "node" for our user-facing (i.e. not authoring) side:
```suggestion
            : "There's currently no content here."
```